livelossplot/input_plugins/generic_keras.py

```diff
@@ -58,9 +56,9 @@ def on_train_begin(self, logs={}):
                 metric2printable_updated['{}_loss'.format(output_name)] =\
                     "{} ({})".format(metric2printable_updated.get(loss_name, loss_name), output_name)
 
-        self.liveplot.metric2title = metric2printable_updated
-        self.liveplot.set_max_epoch(self.params['epochs'])
+        self.matplotlib_output.metric2title = metric2printable_updated
```

Comment:
It is the old way of doing things, but was proved to be cumbersome (as we need to use such things for many other output functionalities).

I suggest dealing with ith at the level of MainLogger.

Reply:
ok, so we need much simplified keras callback :D